src/malcolm/actions/method.actions.js

```diff
@@ -13,6 +16,16 @@ export const buildMethodUpdate = (id, data) => ({
   },
 });
 
+export const malcolmUpdateMethodInput = (path, name, value) => ({
```

Comment:
Should the previous declaration of this in ../malcolmActionCreators.js be deleted?

Reply:
good spot, I'll do that